### PR TITLE
Fsss view ammend count and device type

### PR DESCRIPTION
**Pull request by Felixim0**
Added 2 new attributes to show in the fsss view. Devices will show all the device types and hence has slightly more code than just c/p

This also fixes the IP issues thrown when 200 request not received

### CODE REVIEW

File: app/employeehandler.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ async def get(self, request):
         if get_employee_info.status_code != 200:
             logger.warn(
                 'Attempted to login with invalid user name and/or password',
-                client_ip=request['client_ip'])
+                client_ip=request.get('client_ip', None))
```
Comment (garethTurner) [inline app/employeehandler.py:52]: is this change needed or was this just while you were trying to fix something

File: app/views/fsss_view.py [hunk 2]
```diff
@@ -2,7 +2,12 @@
 from app.tabutils import tab_generation, table_generation, format_to_uk_dates
 from app.fieldmapping import map_employee_name
 
-
+def get_device_types(devices):
+    types_of_device = ""
+    if devices != []:
+        for device in devices:
+            types_of_device = types_of_device + str(device['Device ID']) + "\n"
```
Comment (garethTurner) [inline app/views/fsss_view.py]: this is called types of device but it looks like it will just be a list of the device IDs, is this right?

File: app/views/fsss_view.py [hunk 3]
```diff
@@ -15,8 +20,13 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
  
     devices, device_numbers = process_device_details(device_information)
+#   raise Exception(str(devices))
```
Comment (garethTurner) [inline app/views/fsss_view.py]: should these be commented out?

File: app/views/fsss_view.py (hunk 2 above)
Comment (garethTurner) [reply app/views/fsss_view.py]: or does `device['Device ID']` return the whole device row?

File: app/employeehandler.py (hunk 1 above)
Comment (Felixim0) [reply app/employeehandler.py:52]: Yeah, it throws a key error without this change (but only runs when the service gives a non 200 response, so was missed when the other similar lines were changed)

File: app/views/fsss_view.py (hunk 2 above)
Comment (Felixim0) [reply app/views/fsss_view.py]: Yes this is wrong! I`ll update it to device['Device Type']

File: app/views/fsss_view.py (hunk 3 above)
Comment (Felixim0) [reply app/views/fsss_view.py]: Yes but I`ll delete them now (they were just so I could see  the value of the "devices" var

**Review by garethTurner — APPROVED** [review]
